backend/pyproject.toml

```diff
@@ -190,7 +191,7 @@ dependencies = [
     "drf-jwt<2.0.0,>=1.19.2",
     "drf-spectacular[sidecar]<1.0,>=0.27",
     "drf-extensions==0.7.1",
-    "elasticsearch-dsl<8.0.0,>=7.0.0",
+    "elasticsearch-dsl==8.13.0",
```

Comment:
do we want to have <9.0 or <8.14? @KonradMarzec1991

Reply:
Prefer <8.14